include/circt/Dialect/SV/SVExpressions.td

```diff
@@ -121,3 +123,38 @@ def LocalParamOp : SVOp<"localparam",
 
   let verifier = "return ::verify$cppClass(*this);";
 }
+
+def IndexedPartSelectOp
```

Comment:
I'm sorry for the delay getting back to this, but did you consider putting this op in the comb dialect?  Is this equivalent to `ArraySliceOp` there?

The inout version of this seems necessary, but this op seems quite similar.

Reply:
Thanks Chris, I missed the `ArraySliceOp`, yes it is the same op. I will try to merge them.

Reply:
Thanks Chris, I missed the `ArraySliceOp`, it is the same op. I will try to merge them.